src/index.js

```diff
@@ -44,6 +44,27 @@ export default class Kinto {
   }
 
   /**
+   * Creates a remote transformer constructor, the ES5 way.
+   *
+   * @return {RemoteTransformer}
+   */
+  static createRemoteTransformer(proto) {
+    if (!proto || typeof proto !== "object")
+      throw new Error("Expected prototype object.");
```

Comment:
Bah, this must have changed recently, I remember we conformed to it for
Loop.

Still, could you please find a distinct issue about this?

Nicolas Perriault
Le 31 août 2015 7:31 PM, "Alexis Metaireau" <notifications@github.com> a
écrit :

> In src/index.js
> <https://github.com/Kinto/kinto.js/pull/139#discussion_r38335944>:
>
> > @@ -44,6 +44,27 @@ export default class Kinto {
> >    }
> >
> >    /**
> > +   * Creates a remote transformer constructor, the ES5 way.
> > +   *
> > +   * @return {RemoteTransformer}
> > +   */
> > +  static createRemoteTransformer(proto) {
> > +    if (!proto || typeof proto !== "object")
> > +      throw new Error("Expected prototype object.");
>
> They actually state that you should always use them:
>
> Always brace controlled statements, even a single-line consequent of an if
> else else. This is redundant typically, but it avoids dangling else bugs,
> so it's safer at scale than fine-tuning.
> — Developer Coding Style guide
> <https://developer.mozilla.org/en-US/docs/Mozilla/Developer_guide/Coding_Style#Control_Structures>
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Kinto/kinto.js/pull/139/files#r38335944>.
>